ecl/hthor/hthor.cpp

```diff
@@ -1129,7 +1129,18 @@ void CHThorIndexWriteActivity::execute()
         Owned<IPropertyTree> metadata;
         buildUserMetadata(metadata);
         buildLayoutMetadata(metadata);
-        unsigned nodeSize = metadata ? metadata->getPropInt("_nodeSize", NODESIZE) : NODESIZE;
+        unsigned nodeSize = NODESIZE;
+        if (metadata)
+        {
```

Comment:
This code should really be common rather than duplicated, but it will probably get refactored up in the future.